### DISCUSSION TITLE
EQ out a specific frequency

### DISCUSSION THREAD

**Discussion by mattgt933** [pschatzmann/ESP32-A2DP#455] [Q&A] [answered]
How would I remove a small range of frequencies to get rid of resonance?

**Comment by pschatzmann** [comment, marked as answer]
You could try to apply a filter!

The Readme points you to the project that contains all the relevant information.

**Comment by mattgt933** [comment]
Ok, thanks for the suggestion! I was also looking at the EQ function and it was looking like I could just change the range of the mid section and then control the volume that way.
